### PR TITLE
Fix PHP 5.3 incompatibilities

### PR DESCRIPTION
**Pull request by drichelson**
This is now working with PHP 5.3.25

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 machine:
   php:
-    version: 5.4.37
+    version: 5.3.25
```
Comment (drichelson) [inline circle.yml]: This is the latest php version that CircleCI gives us.

File: circle.yml [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
 
 test:
   override:
+    # syntax check php files
+    - find ./src -name "*.php" -type f -print0 | xargs -0 -I {} php -l {}
```
Comment (drichelson) [inline circle.yml]: 'compile' step to ensure php 5.3 compatibility in case tests don't have 100% code coverage

File: composer.json [hunk 3]
```diff
@@ -20,9 +20,10 @@
         }
     ],
     "require": {
-        "php": ">=5.3",
+        "php": "5.3.*",
         "guzzle/guzzle": "dev-master#ecb935d2d0ecd8cddae4dcfb90515cac5e2cb023",
-        "psr/log": "1.0.0"
+        "psr/log": "1.0.0",
+        "justinrainbow/json-schema": "v1.6.0"
```
Comment (drichelson) [inline composer.json]: This is a transitive dependency, but the version is not pinned, and by default composer will bring in a version that is too high to be compatible with 5.3.25. This is the highest version compatible with 5.3.25

**Comment by jkodumal** [comment]
👍

**Pull request merged**